lms/static/scripts/frontend_apps/components/dashboard/DashboardBreadcrumbs.tsx

```diff
@@ -0,0 +1,62 @@
+import {
+  ArrowLeftIcon,
+  CaretRightIcon,
+  Link,
+} from '@hypothesis/frontend-shared';
+import classnames from 'classnames';
+import { Link as RouterLink } from 'wouter-preact';
+
+type BreadcrumbLink = {
```

Comment:
This type should be exported since it features in exported APIs. Ideally we should have a lint for this.

Reply:
Ugh, right.